### PR TITLE
Listings in progress graphql

### PR DESCRIPTION
**Pull request by nickreynolds**
- create new Listings In Progress Container that renders different components based on whether or not graphql is being used
- TODO: cleanup query/components to pass actual listing data to ListingList components instead of just addresses, since we already have all the data we need we shouldn't be making extra queries

### CODE REVIEW

File: packages/dapp/src/components/listinglist/ListingsInProgressRedux.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  awaitingAppealChallengeListings: Set<string>;
+  appealChallengeCommitPhaseListings: Set<string>;
+  appealChallengeRevealPhaseListings: Set<string>;
+  resolveChallengeListings: Set<string>;
+  resolveAppealListings: Set<string>;
+}
+
+class ListingsInProgressRedux extends React.Component<ListingProps & ListingReduxProps> {
+  public render(): JSX.Element {
+    console.log("Listings in Progress Redux");
```
Comment (am9u) [inline packages/dapp/src/components/listinglist/ListingsInProgressRedux.tsx]: extra `console.log`?

File: packages/dapp/src/helpers/queryTransformations.ts [hunk 2]
```diff
@@ -95,16 +121,47 @@ export function transformGraphQLDataIntoChallenge(queryChallengeData: any): Chal
...
+  } else {
+    return undefined;
+  }
+}
+
+export function transformGraphQLDataIntoAppeal(queryAppealData: any): AppealData | undefined {
+  if (queryAppealData) {
+    return {
+      requester: queryAppealData.requester,
+      appealFeePaid: new BigNumber(10), // queryAppealData.appealFeePaid
```
Comment (am9u) [inline packages/dapp/src/helpers/queryTransformations.ts]: Is field just missing from the GraphQL response / the crawler isn't picking it up yet?

File: packages/dapp/src/components/listinglist/ListingsInProgressContainer.tsx [hunk 3]
```diff
@@ -124,14 +69,19 @@ class ListingsInProgressContainer extends React.Component<
               return <p>Error :</p>;
             }
             const map = Set<any>(data.listings);
-            const allListings = map.map(listing => {
-              const transformed = transformGraphQLDataIntoListing(listing, listing!.contractAddress);
-              return transformed;
-            });
+            const allListings: Set<NewsroomListing> = map
+              .map(listing => {
+                console.log("listing: ", listing);
```
Comment (am9u) [inline packages/dapp/src/components/listinglist/ListingsInProgressContainer.tsx]: extra `console.log`?

File: packages/dapp/src/components/listinglist/RejectedListingListContainer.tsx [hunk 4]
```diff
@@ -31,9 +38,13 @@ class RejectedListingListContainer extends React.Component<RejectedListingsListC
             if (error) {
               return <p>Error :</p>;
             }
-            const map = Set<string>(
+            const map = Set<NewsroomListing>(
               data.listings.map((listing: any) => {
-                return listing.contractAddress.toLowerCase();
+                console.log("listing: ", listing);
```
Comment (am9u) [inline packages/dapp/src/components/listinglist/RejectedListingListContainer.tsx]: extra `console.log`?

File: packages/dapp/src/helpers/queryTransformations.ts [hunk 5]
```diff
@@ -107,7 +114,7 @@ export function transformGraphQLDataIntoListing(listing: any, listingAddress: st
 export function transformGraphQLDataIntoChallenge(queryChallengeData: any): ChallengeData | undefined {
   if (queryChallengeData) {
     return {
-      statement: queryChallengeData.statement,
+      statement: "",
```
Comment (am9u) [inline packages/dapp/src/helpers/queryTransformations.ts]: Is the statement not currently in the GraphQL response? It looks like it's specified in `LISTING_FRAGMENT`...

Comment (nickreynolds) [reply packages/dapp/src/helpers/queryTransformations.ts]: going to be dealing with that in my next PR, it was coming down in a way that broke rendering

Comment (am9u) [reply packages/dapp/src/helpers/queryTransformations.ts]: 👍

**Pull request merged**